src/libcrun/cgroup-utils.c

```diff
@@ -71,6 +71,41 @@ libcrun_cgroups_create_symlinks (int dirfd, libcrun_error_t *err)
   return 0;
 }
 
+int
+make_cgroup_threaded (const char *path, libcrun_error_t *err)
+{
+  cleanup_free char *cgroup_path_type = NULL;
+  int ret;
+
+  path = consume_slashes (path);
+
+  if (path == NULL || path[0] == '\0')
+    return 0;
+
+  ret = append_paths (&cgroup_path_type, err, CGROUP_ROOT, path, "cgroup.type", NULL);
+  if (UNLIKELY (ret < 0))
+    return ret;
+
+  ret = write_file (cgroup_path_type, "threaded", strlen ("threaded"), err);
```

Comment:
```suggestion
  ret = write_file (cgroup_path_type, "threaded", 8, err);
```

Should we mark len as 8 since `threaded` is a constant and not expected to change.

Reply:
I think it is just easier to read, I'll make it a const